### ISSUE TITLE
Changelog not accurate

### ISSUE THREAD

**Issue by ManiacOfGitHub** [SciresM/boot9strap#36]
These lines specifically in the changelog are incorrect.
> If the ntrboot combo (X+Start+Select) is held, or if either FIRM file is corrupt, the notification LED will display the following:
> - SD FIRM corrupt, CTRNAND FIRM also corrupt: red

> "if either FIRM file is corrupt"

If the CTRNAND FIRM is corrupt but the SD FIRM is successfully loaded, as long as no ntrboot combo is held, it will not show an LED. Also, if both FIRM files are missing, it will show the white LED even if the combo isn't held.

> "CTRNAND FIRM also corrupt"

This also applies if CTRNAND FIRM is missing.

<ins>An example of a revised version:</ins>

If the ntrboot combo (X+Start+Select) is held, if the SD FIRM is corrupted or neither FIRM files can be successfully loaded, the notification LED will display the following:
- SD FIRM successfully loaded: green
- SD FIRM missing, CTRNAND FIRM successfully loaded: yellow
- SD FIRM corrupt, CTRNAND FIRM successfully loaded: orange
- SD FIRM missing, CTRNAND FIRM also missing: white
- SD FIRM missing, CTRNAND FIRM corrupt: magenta
- SD FIRM corrupt, CTRNAND FIRM corrupt/missing: red
- in addition to the above, the LED will blink if it is actually a ntrboot boot

(Keep in mind this is just an example of what it could be. It's probably worded badly.)

**Comment by skunkmommy** [comment]
The original is fine. You're arguing semantics. Your revision is too specific making it longer and therefore harder to read.

There should probably be an LED colour for missing/corrupt CTRNAND firm but successfully loaded SD FIRM.
A workaround would be to rename the boot.firm on your SD so it thinks it's missing and then diagnose, shouldn't have to but it works.
I assume it tries to boot from SD FIRM first, which makes sense as that's most likely to be up to date since it's literally just a file on the root of the card that you can replace whenever.

**Comment by ManiacOfGitHub** [comment]
> There should probably be an LED color for missing/corrupt CTRNAND firm but successfully loaded SD FIRM.

I agree. Why not also do the same thing when the SD FIRM is corrupt and the CTRNAND FIRM is missing? Then, there would be a color for all 9 combinations of FIRM statuses.
>Your revision is too specific making it longer and therefore harder to read.

Why not make it a table so it gives more information in a more digestible format?

|                          | **SD FIRM Loaded** | **SD FIRM Missing** | **SD FIRM Corrupt** |
|--------------------------|--------------------|---------------------|---------------------|
| **CTRNAND FIRM Loaded**  | Green              | Yellow              | Orange              |
| **CTRNAND FIRM Missing** | Green (change)     | White               | Red (change)        |
| **CTRNAND FIRM Corrupt** | Green (change)     | Magenta             | Red                 |
